### ISSUE TITLE
Github Visualizer

### ISSUE THREAD

**Issue by elisebigley** [digh5000/student-tutorials#25]
Good tutorial! I think it would be helpful if you had the link to the GitHub Visualizer website so we could access it from the tutorial page. For someone like me... who is not that techno-savvy it would be helpful if you would explain what you mean by language in your tutorial ;) Other than that the instructions were really easy to follow and it was cool to see my data visualized!

**Comment by SAllieW** [comment]
I agree with Elise - very cool! Awesome job displaying everything with screenshots. I think the link to the first image might be broken? The other images and links are dandy tho
